test/client-checkout-integration-test.js

```diff
@@ -229,6 +230,68 @@ suite('client-checkout-integration-test', () => {
     });
   });
 
+  test('it resolves with a checkout on Client.checkout#updateShippingAddress', () => {
+    const checkoutId = checkoutShippingAddressUpdateFixture.data.checkoutShippingAddressUpdateV2.checkout.id;
+    const shippingAddress = {
```

Comment:
Is there a way to reuse this for the test below?

Reply:
Done